### ISSUE TITLE
Calico-node fails to start intermittently

### ISSUE THREAD

**Issue by alexbrand** [projectcalico/calico-containers#1284]
We have been noticing that Calico sometimes fails to start with the following error:

```
Nov 07 15:41:14 ip-10-0-3-34 systemd[1]: Starting Calico per-node agent...
Nov 07 15:41:15 ip-10-0-3-34 calicoctl[19985]: Running Docker container with the following command:
Nov 07 15:41:15 ip-10-0-3-34 calicoctl[19985]: docker run --restart=always --net=host --privileged --name=calico-node -e HOSTNAME=ip-10-0-3-34 -e IP=10.0.3.34 -e IP6= -e CALICO_N
Nov 07 15:41:15 ip-10-0-3-34 calicoctl[19985]: Calico node is running with id: f987c2bbecb03d322ace93222ff43643a7e7e2d73b7fc237a4526959026b8f56
Nov 07 15:41:15 ip-10-0-3-34 calicoctl[19985]: Waiting for successful startup
Nov 07 15:41:15 ip-10-0-3-34 calicoctl[19985]: Waiting for etcd connection...
Nov 07 15:41:15 ip-10-0-3-34 calicoctl[19985]: Traceback (most recent call last):
Nov 07 15:41:15 ip-10-0-3-34 calicoctl[19985]: File "startup.py", line 336, in <module>
Nov 07 15:41:15 ip-10-0-3-34 calicoctl[19985]: main()
Nov 07 15:41:15 ip-10-0-3-34 calicoctl[19985]: File "startup.py", line 309, in main
Nov 07 15:41:15 ip-10-0-3-34 calicoctl[19985]: client.ensure_global_config()
Nov 07 15:41:15 ip-10-0-3-34 calicoctl[19985]: File "/usr/lib/python2.7/site-packages/pycalico/datastore.py", line 130, in wrapped
Nov 07 15:41:15 ip-10-0-3-34 calicoctl[19985]: return fn(*args, **kwargs)
Nov 07 15:41:15 ip-10-0-3-34 calicoctl[19985]: File "/usr/lib/python2.7/site-packages/pycalico/datastore.py", line 296, in ensure_global_config
Nov 07 15:41:15 ip-10-0-3-34 calicoctl[19985]: self._ensure_cluster_guid(CLUSTER_GUID_PATH)
Nov 07 15:41:15 ip-10-0-3-34 calicoctl[19985]: File "/usr/lib/python2.7/site-packages/pycalico/datastore.py", line 314, in _ensure_cluster_guid
Nov 07 15:41:15 ip-10-0-3-34 calicoctl[19985]: except EtcdAlreadyExist:
Nov 07 15:41:15 ip-10-0-3-34 calicoctl[19985]: NameError: global name 'EtcdAlreadyExist' is not defined
Nov 07 15:41:15 ip-10-0-3-34 calicoctl[19985]: Calico node failed to start
Nov 07 15:41:15 ip-10-0-3-34 systemd[1]: calico-node.service: main process exited, code=exited, status=1/FAILURE
Nov 07 15:41:15 ip-10-0-3-34 systemd[1]: Unit calico-node.service entered failed state.
Nov 07 15:41:15 ip-10-0-3-34 systemd[1]: calico-node.service failed.
```

Image version: `calico/node:v0.22.0`

Command line flags:
```
/usr/bin/calicoctl node --ip=10.0.3.34 --node-image=calico/node:v0.22.0 --no-pull --detach=false
```

**Comment by caseydavenport** [comment]
@alexbrand Sounds to me like you're hitting a similar issue to this: https://github.com/projectcalico/libcalico/issues/156 which was fixed by https://github.com/projectcalico/libcalico/pull/155, but wasn't included until `calico/node:v0.23.0`.

Would you be able to upgrade to the newer release? That should fix up the problem.

**Comment by caseydavenport** [comment]
Going to close this guy - let me know if you're still hitting issues!
